src/Microsoft.Health.Functions.Extensions/Microsoft.Health.Functions.Extensions.csproj

```diff
@@ -21,6 +22,8 @@
     <PackageReference Include="Microsoft.Extensions.Logging" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" />
     <PackageReference Include="Newtonsoft.Json" />
+    <PackageReference Include="NSubstitute" />
+    <PackageReference Include="System.Drawing.Common" />
```

Comment:
Do you really need `System.Drawing.Common`? Presumably this is a build-time dependency for some component governance alert? Could we use `PrivateAssets="All"`?